doc/content/integrations/lora-cloud/das.md

```diff
@@ -0,0 +1,107 @@
+
+LoRa Cloud Device & Application Services protocol lets you manage common device functionality at the application layer for all LoRa-based devices, and manage geolocation for modem-based devices (like [LoRa Edge™ LR1110 Solution](https://www.loracloud.com/documentation/device_management?url=overview.html#lora-edge-lr1110-solution)). These services also provide protocols for reliable application data reception from all Semtech's LoRaWAN-compatible devices, with efficient techniques to fight against packet loss. 
+
+{{< note >}} In general, LoRa Cloud Device & Application Services do not interact directly with network servers to retrieve uplinks or schedule downlinks, so it is usually client's responsibility to forward uplinks and schedule downlinks as a part of the response.
+
+However, when it comes to geolocation for modem-based devices, {{% tts %}} integration with LoRa Cloud Device & Application Services eliminates this need, i.e. it queries the protocol engine on LoRa Cloud in order to retrieve the response. This way, after the integration is configured and the uplink message containing scanning result is sent from the device, the user only has to monitor results in {{% tts %}}. {{</ note >}}
+
+## LoRa Cloud Setup
+
+On the [LoRa Cloud portal](https://www.loracloud.com/portal), navigate to **LoRa Cloud Device & Application Services**.
+
+{{< figure src="../das.png" alt="LoRa Cloud Device & Application Services" >}}
+
+Choose **MANAGE DEVICES** on the left hand menu and click the **AD DEVICES** button. To add a device, you only need to provide its EUI as shown on the image below.
+
+{{< figure src="../add-device.png" alt="Adding a new device" >}}
+
+Next, choose **MANAGE TOKENS** on the left hand menu. Click the **CREATE NEW TOKEN** button.
+
+Give the name to your token and tick the boxes to allow all listed permissions. Click the **CREATE NEW TOKEN** button to finish.
+
+{{< figure src="../token-creation.png" alt="Creating a token" >}}
+
+## Configure Device & Application Services Integration
+
+To setup LoRa Cloud Device & Application Services integration, navigate to your application in {{% tts %}}.
+
+On the left hand menu, select **Integrations &#8594; LoRa Cloud**. Click the **Expand** button next to the **Device & Application Services**.
+
+Next, copy the token you previously created in LoRa Cloud Device & Application Services portal.
+
+{{< figure src="../copy-token.png" alt="Copy the token from LoRa Cloud" >}}
+
+Paste the token in the **Token** field under the **Set LoRa Cloud token** section and tick the box under **LR1110 demo encoding** to enable it. Finish by clicking the **Set token** button.
+
+{{< figure src="../paste-token.png" alt="Paste the token in The Things Stack" >}}
+
+## Obtain Location Solutions 
+
+In general, location of the device can be estimated using Wi-Fi, assisted GNSS and autonomous GNSS scans. This section will show you how to obtain the location estimates using Wi-Fi and assisted GNSS scans.
```

Comment:
```suggestion
For LoRa Edge capable devices, location of the device can be estimated using Wi-Fi, assisted GNSS and autonomous GNSS scans. This section will show you how to obtain the location estimates using Wi-Fi and assisted GNSS scans.
```

Reply:
Maybe also add a note here that this is NOT LoRa Cloud Geolocation, with a reference to Geolocation, if they look for RSSI/TDOA solutions?